specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -128,6 +128,9 @@
         "x-ms-examples": {
           "StorageAccountCreate": {
             "$ref": "./examples/StorageAccountCreate.json"
+          },
+          "StorageAccountCreateUserAssignedEncryptionIdentityWithCMK": {
```

Comment:
Does UserAssignedEncrption only work for PUT？

Reply:
It works for both PUT/PATCH.